<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsWindow</class>
 <widget class="QDialog" name="settingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioButton_matrix">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>229</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 208px;
height: 46px;
left: 720px;
top: 246px;
border: solid;
font-family: 'Mountains of Christmas';
font-style: normal;
font-weight: 400;
font-size: 32px;
line-height: 44px;
letter-spacing: 0.24em;

color: #FFFFFF;
</string>
   </property>
   <property name="text">
    <string>Matrix</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>131</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 181px;
height: 41px;
left: 1089px;
top: 375px;

font-family: 'Mountains of Christmas';
font-style: normal;
font-weight: 400;
font-size: 32px;
line-height: 44px;
letter-spacing: 0.24em;

color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>Epochs</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selection Part</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_graph">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>233</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 208px;
height: 46px;
left: 720px;
top: 246px;

font-family: 'Mountains of Christmas';
font-style: normal;
font-weight: 400;
font-size: 32px;
line-height: 44px;
letter-spacing: 0.24em;
text-align:center;

color: #FFFFFF;
</string>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="kgroupSpin">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>370</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>K-Group</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>117</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 181px;
height: 41px;
left: 1089px;
top: 375px;

font-family: 'Mountains of Christmas';
font-style: normal;
font-weight: 400;
font-size: 32px;
line-height: 44px;
letter-spacing: 0.24em;

color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>Layers</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="crossValidationCheck">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cross Validation</string>
   </property>
  </widget>
  <widget class="QComboBox" name="layers_comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>103</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="epochs_comboBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
